tokio/src/task/mod.rs

```diff
@@ -279,6 +279,7 @@ cfg_rt! {
     pub use crate::runtime::task::{JoinError, JoinHandle};
 
     mod blocking;
+    #[cfg(not(target_os = "wasi"))]
     pub use blocking::spawn_blocking;
```

Comment:
This should probably mark `block_in_place` too.

Reply:
`block_in_place` is wrapped by `cfg_rt_multi_thread` which already excludes WebAssembly.